modules/account/forms/profile_modify.form.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form xmlns="http://jelix.org/ns/forms/1.1">
+
+<input type="string" ref="name">
+    <label locale="account~account.profile.th.name"/>
+</input>
+
+<input type="email" ref="email">
+    <label locale="account~account.profile.th.email"/>
+</input>
+
+<input type="string" ref="provider">
+    <label locale="account~account.profile.th.provider"/>
+</input>
+
+<datetime ref="create_date">
+    <label locale="account~account.profile.th.createdate"/>
+</datetime>
+
+<menulist ref="status">
```

Comment:
should be an output, but we should find a way to display the label to the corresponding value, instead of the value.